### ISSUE TITLE
[BATX on T460] Some minor issues and a feature request.

### ISSUE THREAD

**Issue by junaedahmed** [benbender/x1c6-hackintosh#11]
Hello @benbender ,

First of all many thanks for your hard work on totally acpi based battery ssdt implementation.
I recently found that you implemented dual battery support. Upon trying it on my T460 I am having some problem with it.

- In BATX you used GBST as one of the method name. In T460 DSDT there is already a GBST method present so it conflicts with that, I think CBST for that method name will be good for all supported devices.
- I have noticed Time to full charge is broken with this ssdt. It showing 29m to full charge constantly for all the way to full charge. Did X1 have this problem too?
- ~Battery temperature is constant 30 degree C no matter what. Is that read correctly on yours end?~
- And lastly a feature request, as you might have noticed in hackintoshes low battery sleep (hibernate) doesn't really work, it shows the notification but keeps running on until the battery empties. below is a workaround to do a low battery sleep from ssdt. If you are interested have a look.
[Hibernate on low battery](https://applelife.ru/threads/hibernate-pri-razrjade-batarei.2874421/)

Regards,
Junaed

**Comment by velaar** [comment]
> * Battery temperature is constant 30 degree C no matter what. Is that read correctly on yours end?

That is a fake sensor. I don't think there is a battery temperature sensor in X1C6 (at at least accessible through IC)

**Comment by junaedahmed** [comment]
@velaar ,
Thanks for confirming that. could please also check on the charging time issue I have mentioned. Is charging time correctly reported on X1?

**Comment by mhl221135** [comment]
>@junaedahmed  

Do you use BATX on your efi ? can you update your efi folder please if you use

**Comment by junaedahmed** [comment]
@mhl221135 yes currently I am using BATX. It's quite easy just removed previous patches and renamed the conflicted method names in BATX. I am planning to update my EFI, but this month I am stumped. I will definitely update it in February.
